.github/workflows/format.yml

```
@@ -13,10 +13,13 @@ jobs:
       - uses: actions/checkout@v3
         with:
           token: ${{ secrets.USERSTYLES_TOKEN }}
+      - uses: actions/setup-node@v3
+
       - name: Run Prettier
-        uses: creyD/prettier_action@v4.3
+        run: npx prettier --write styles/**/*.css
+
+      - name: Commit changes
+        uses: EndBug/add-and-commit@v9
         with:
-          commit_options: "--author 'github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>'"
-          commit_message: "style: format code with prettier"
-          prettier_options: "--write styles/**/*.css"
-          prettier_version: "2.8.4"
+          default_author: github_actions
+          message: "style: format code with prettier"
```

I'd appreciate it, if we could use the same action as we do for generate.yml:
```suggestion
      - name: Push Changes
        uses: stefanzweifel/git-auto-commit-action@v4
        with:
          commit_message: "style: format code with prettier"
          commit_author: "github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>"
```